tools/props/Versions.props

```diff
@@ -27,8 +27,8 @@
   </PropertyGroup>
   <!-- NetFx and NetCore project dependencies -->
   <PropertyGroup>
-    <AzureIdentityVersion>1.10.3</AzureIdentityVersion>
-    <MicrosoftIdentityClientVersion>4.56.0</MicrosoftIdentityClientVersion>
+    <AzureIdentityVersion>1.11.0</AzureIdentityVersion>
+    <MicrosoftIdentityClientVersion>4.60.1</MicrosoftIdentityClientVersion>
```

Comment:
Time to bump these already... 👀 

https://www.nuget.org/packages/Microsoft.Identity.Client/#versions-body-tab